hoot-core-test/src/test/cpp/hoot/core/visitors/SplitLongLinearWaysVisitorTest.cpp

```diff
@@ -86,8 +86,9 @@ class SplitLongLinearWaysVisitorTest : public HootTestFixture
       for ( unsigned int i = startNodeId; i <= endNodeId; i++ )
       {
         // Make sure map does not already have node in question
-        if ( _map->containsNode(i) == false ) {
-          boost::shared_ptr<Node> createNode(new Node(Status::Unknown1, i, i * 1.0, i * 1.0, 1.0));
+        if ( _map->containsNode(i) == false )
+        {
+          std::shared_ptr<Node> createNode(new Node(Status::Unknown1, i, i * 1.0, i * 1.0, 1.0));
```

Comment:
Took a little time to find all instances of `{` on the same line as the `if` or `for` etc. and move it down.  This isn't Java :coffee: you know!